src/sagemaker/chainer/model.py

```diff
@@ -99,12 +99,9 @@ def __init__(
                 file which should be executed as the entry point to model
                 hosting. If ``source_dir`` is specified, then ``entry_point``
                 must point to a file located at the root of ``source_dir``.
-            image_uri (str): A Docker image URI (default: None). In serverless
-                inferece, it is required. More image information can be found in
-                `Amazon SageMaker provided algorithms and Deep Learning Containers
-                <https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-algo-docker-registry-paths.html>`_.
-                In instance based inference, if not specified, a default image for
-                Chainer will be used. If ``framework_version`` or ``py_version``
+            image_uri (str): A Docker image URI (default: None). If not specified,
+                a default image for Chainer will be used.
+                If ``framework_version`` or ``py_version``
                 are ``None``, then ``image_uri`` is required. If also ``None``,
```

Comment:
please update what's none in "If also ``None``"

Reply:
This is the original docstring statement we have - [API Version 2.78](https://sagemaker.readthedocs.io/en/v2.78.0/frameworks/chainer/sagemaker.chainer.html#chainer-model). Do you think we need to change this? Or may just keeping the original statement?

Reply:
Let's change it as it will help later :)